### PR TITLE
Configure IPA with useful inspection settings

### PR DESCRIPTION
**Pull request by priteau**
Add configuration to include extra-hardware and mellanox elements, with other useful collectors and kernel settings.

### CODE REVIEW

File: etc/kayobe/ipa.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # Ironic Python Agent (IPA) image build configuration.
 
 # Whether to build IPA images from source.
-#ipa_build_images:
+ipa_build_images: True
```
Comment (markgoddard) [inline etc/kayobe/ipa.yml]: This does change the workflow a bit - now you need to build an image, both for bifrost & overcloud ironic. I'd at least avoid putting this in wallaby.

My medium term goal is to build IPA images and put them on Ark, then just include config to pull them down in here. The build flag would go in the `ci-builder` environment.

**Comment by markgoddard** [comment]
What should we do here?

**Comment by priteau** [comment]
> What should we do here?

Are we still planning to build IPA images and host them on Ark?

**Comment by markgoddard** [comment]
> > What should we do here?
> 
> Are we still planning to build IPA images and host them on Ark?

That is in the plan, although not scheduled in the near future.

**Comment by priteau** [comment]
> > > What should we do here?
> > 
> > 
> > Are we still planning to build IPA images and host them on Ark?
> 
> That is in the plan, although not scheduled in the near future.

Thanks. I need to revisit this because IPA images on recent releases will use CentOS Stream 9.

File: etc/kayobe/ipa.yml (hunk 1 above)
Comment (priteau) [reply etc/kayobe/ipa.yml]: I removed `ipa_build_images` for now, it is up to the operator to enable it.

**Review by MoteHue — DISMISSED** [review]
LGTM

File: etc/kayobe/ipa.yml [hunk 2]
```diff
@@ -134,6 +139,10 @@
 
 # List of additional kernel parameters for Ironic python agent.
 #ipa_kernel_options_extra:
```
Comment (markgoddard) [inline etc/kayobe/ipa.yml]: Need to uncomment?

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/kayobe/ipa.yml (hunk 2 above)
Comment (priteau) [reply etc/kayobe/ipa.yml]: D'oh!

**Review by markgoddard — APPROVED** [review]

**Review by MoteHue — APPROVED** [review]

**Review by MoteHue — APPROVED** [review]

**Comment by Alex-Welsh** [comment]
I've resolved the final conversation so this is now mergable, but it was last approved over a month ago. @priteau or @MoteHue could you quickly check it's not stale since then, and click the button?

**Comment by MoteHue** [comment]
> I've resolved the final conversation so this is now mergable, but it was last approved over a month ago. @priteau or @MoteHue could you quickly check it's not stale since then, and click the button?

Still LGTM, let's merge :)

**Pull request merged**